_posts/2017-03-14-pycon-pune-2017-a-wonderful-python-experience.md

```diff
@@ -0,0 +1,220 @@
+
+Because I did the huge amount of traveling at that night, I was unable to get
+enough sleep. Such tiredness resulted as an eye infection. I managed to attend
+the first day of the conference, but I was not in a condition to attend the
+second day. Treatment of local doctor healed me 2 days and then I was able to
+take part into Dev sprints.
+
+### Conference
+
+The conference was of total 4 days where initial two days were for the talks and
+end was assigned for a Devsprint. It didn't bombard with many tracks but gave
+quality talks presented in a single track. Timings were from 9 AM to 5 PM which
+was taken little lightly by the attendees.
+
+I was pretty impressed with the keynote speakers of this conference.
+
+#### [Katie Cunningham](https://twitter.com/kcunning)
+
+Katie is the O'Reilly author. Her
+[book](http://shop.oreilly.com/product/0636920024514.do) on Accessibility
+depicts her area of expertise. She is fun to talk. She likes to listen about
+developer communities, writing and most importantly computer games. Her broad
+vision on product development is amazing. She is an avid reader. I enjoyed
+listening to her experience of being in India very first time.
+
+#### [Honza Kral](https://twitter.com/honzakral)
+
+Honza is the dude who loves contributing to
+[Django](https://www.djangoproject.com/). He is the core contributor of Django
+too. He hacks on Python drivers at the [Elastic](https://www.elastic.co/). I was
+impressed with his suggestions on code design problem I was trying to solve from
+past few months.  His suggestions on code design are worth noticing. He is
+Vimmer and maintains little [vim](http://vim.org) plugins as a part of his
+interest.
+
+#### [Stephen J. Turnbull](https://twitter.com/yasegumi)
+
+Stephen professes the Dismal Science of Economics. His knowledge is deep-rooted
+just like his beard. You will enjoy discussing on Computer science, books and
+his experience of programming. He is authoring few books written in the Japanese
+language. Stephen is [Emacsite](https://www.gnu.org/s/emacs/).
+
+![stephen_turnbull]({{
+site.url}}/assets/images/stephen_turnbull.jpg)
+
+#### [Terri Oda](https://twitter.com/terriko)
+
+Terri is a security nerd. She spent most of her time exploring tools at Intel.
+Terri knows how to hide from the spying of the U.S. Government. She is leading
+[Google Summer of Code](https://summerofcode.withgoogle.com/) section from
+[Python Software Foundation](https://www.python.org/psf/). Terri is PSF
+community service award winner. If you are a student and want to take part in
+GSoC choosing Python as your organization then She is the right person to talk.
+
+#### [Florian Fuchs](https://github.com/flofuchs)
+
+His knowledge on [ReST
+API](https://en.wikipedia.org/wiki/Representational_state_transfer) construction
+is the best. He is [Falcon](https://falconframework.org/) nerd too. I enjoyed
+discussing on various authentication mechanisms for ReST API with him.  He is
+the Red hatter.
```

Comment:
Since Florian isn't the *only* person at Red Hat, this should be "a", not "the". "Red Hat" is a company name, so it should have both words capitalized.